tests/openapi/openapi_integration/test_multi_vector.py

```diff
@@ -30,7 +30,7 @@ def multivector_collection_setup(collection_name='test_collection'):
                     "size": 4,
                     "distance": "Dot",
                     "multi_vec_config": {
-                        "max_sim": {}
+                        "comparator": "max_sim"
```

Comment:
naming is hard, feel free to recommend something better for the field name :)

Reply:
technically the name would be `distance` or `similarity` too, but that would be confusing. I cannot think of a better name than `comparator` right now.

Reply:
the `multi_vec_config` also might be better. Rule of thumb: don't use more then one `_`

Reply:
I can propose `multivec_config` to have a single underscore.

Reply:
I am not a super fan of `comparator` either, it feels a bit too close to programming.

I remember @IvanPleshkov proposing `metric` instead of `comparator`.

That's one I could get behind if it gather votes.